### PR TITLE
Use ignite-3 as parent module for ignite-cli-2 module

### PR DESCRIPTION
**Pull request by Pochatkin**
* Fixed problems with project code style
* Fixed java docs build verifications

### CODE REVIEW

File: modules/ignite-cli-2/src/main/java/org/apache/ignite/cli/commands/decorators/core/DecoratorStorage.java [hunk 1]
```diff
@@ -4,14 +4,29 @@
...
 @Singleton
 public class DecoratorStorage {
     private final Map<Class<?>, Decorator<? extends CommandOutput, ? extends TerminalOutput>> commandDecorators = new ConcurrentHashMap<>();
-    
+
+    /**
+     * Register method.
+     *
+     * @param commandClass Class for applying provided decorator.
+     * @param decorator Non-null decorator instance.
```
[the remark this answers is not in the document]
Comment (Pochatkin) [reply modules/ignite-cli-2/src/main/java/org/apache/ignite/cli/commands/decorators/core/DecoratorStorage.java]: This class removed

File: modules/ignite-cli-2/src/main/java/org/apache/ignite/cli/call/configuration/ReadConfigurationCallInput.java [hunk 2]
```diff
@@ -26,20 +26,26 @@ private ReadConfigurationCallInput(String nodeId, String selector, String cluste
...
+     * Get selector.
+     *
      * @return Selector for configuration tree.
      */
     public String getSelector() {
         return selector;
     }
 
     /**
+     * Get cluster ID.
```
Comment (valepakh) [inline modules/ignite-cli-2/src/main/java/org/apache/ignite/cli/call/configuration/ReadConfigurationCallInput.java:47]: cluster URL?

File: modules/ignite-cli-2/src/main/java/org/apache/ignite/cli/call/configuration/UpdateConfigurationCallInput.java [hunk 3]
```diff
@@ -26,27 +26,35 @@ private UpdateConfigurationCallInput(String nodeId, String config, String cluste
...
+     * Get configuration.
+     *
      * @return Configuration to update.
      */
     public String getConfig() {
         return config;
     }
 
     /**
+     * Get cluster ID.
```
Comment (valepakh) [inline modules/ignite-cli-2/src/main/java/org/apache/ignite/cli/call/configuration/UpdateConfigurationCallInput.java:56]: URL

File: modules/ignite-cli-2/src/main/java/org/apache/ignite/cli/commands/decorators/core/Decorator.java [hunk 4]
```diff
@@ -1,5 +1,17 @@
 package org.apache.ignite.cli.commands.decorators.core;
 
-public interface Decorator<CommandDateType, TerminalDataType extends TerminalOutput> {
-    TerminalDataType decorate(CommandDateType date);
+/**
+ * Interface for transformation command output to terminal output.
```
Comment (valepakh) [inline modules/ignite-cli-2/src/main/java/org/apache/ignite/cli/commands/decorators/core/Decorator.java:4]: for transforming?

File: modules/ignite-cli-2/src/main/java/org/apache/ignite/cli/core/call/DefaultCallOutput.java [hunk 5]
```diff
@@ -50,51 +50,97 @@ public int hashCode() {
...
+    /**
+     * Builder of {@link DefaultCallOutput}.
+     */
     public static class DefaultCallOutputBuilder {
         private CallOutputStatus status;
         private String body;
         private Throwable cause;
 
+        /**
+         * Constructor.
```
Comment (valepakh) [inline modules/ignite-cli-2/src/main/java/org/apache/ignite/cli/core/call/DefaultCallOutput.java:107]: It's not a constructor

**Pull request merged**